src/stylesheets/components/_accordions.scss

```diff
@@ -3,6 +3,33 @@
 $accordion-border: units($theme-accordion-border-width) solid
   color($theme-accordion-border-color);
 
+// Maps for Icons
+$icon-object: (
```

Comment:
Let's rename this to something more specific like `$accordion-icon-defaults-object`.